README.md

```diff
@@ -66,11 +66,11 @@ To include Chimney to your SBT project, add the following line to your `build.sb
 libraryDependencies += "io.scalaland" %% "chimney" % "0.5.3"
 ```
 
-Library is released for Scala 2.11.x, 2.12.x and 2.13.x.
+Library is released for Scala 2.12.x and 2.13.x.
 
 If you want to use it with Scala.js (or Scala Native), you need to replace `%%` with `%%%`.
```

Comment:
It seems Scala native is no longer supported. Maybe delete it mention from README?